server/utils/mojPagination/pageResponse.ts

```diff
@@ -0,0 +1,55 @@
+import { numberRange } from '../utils'
+
+export default class PageResponse {
+  constructor(readonly size: number, readonly pageNumber: number, readonly totalElements: number) {}
```

Comment:
I removed the `content` member variable as it was not used

Reply:
I also removed the firstPage variable, as it would always be 1 (the conversion to/from zero-based is done in pagination.ts)

Reply:
Cool

Reply:
Worth sticking a comment in to the effect that this is 1 based?